### ISSUE TITLE
Improve visibility of side panel state in the admin

### ISSUE THREAD

**Issue by th3hamm0r** [wagtail/wagtail#9174]
### Is your proposal related to a problem?

Currently in the page editor, it is often hard to differentiate between the states "comments on" and "comments off", since it is only shown by the small comment button at the top bar, which uses either dark grey or the primary color.

Yes, there are some other elements which indicate the state, e.g. the notification switch on the top (but on large pages, you have to scroll to the top to see it), or the small "add comment" buttons when you hover a form element, but I think the icon or some other UI should clearly indicate if commenting is enabled.

In this example I've scrolled down a bit on a large page to demonstrate the case, when only the comments icon at the top bar shows the state:
![wagtail_commenting_state_visibility](https://user-images.githubusercontent.com/468164/189330018-e6aa0fa3-de5c-433b-b006-77c9a2c05ea2.gif)
Clicking the icon does not really change that much, beside the small color change 😉

### Describe the solution you'd like

There should be a clear indicator, if commenting is enabled.
While searching for related issues, I've found #8147. If I understand it right, the comment button will then open a visible side panel(?). That could fix this issue too.

**Comment by thibaudcolas** [comment]
Thank you for taking the time to report this @th3hamm0r! This counts as a bug because there isn’t enough color contrast between the two states. And even if there was, changing only the color of an icon can make it pretty hard for people who are color blind to figure out what is happening. And this isn’t just commenting, we need a better representation of the active state for all side panels.

As a worst-case example, here is a screenshot of simulated full achromatopsia. One of those buttons is in its active state:

![wagtaildemo-nightly herokuapp com_admin_pages_77_edit_](https://user-images.githubusercontent.com/877585/189470395-203538e2-7c65-4b9b-b53e-389da53a3ddf.png)

**Comment by Theresa-o** [comment]
Hello @thibaudcolas I would like to work on this.

**Comment by Theresa-o** [comment]
oK @thibaudcolas let me work on this - think of the design, improve the accessibility - and revert.

Thank you for the suggestions.

**Comment by thibaudcolas** [comment]
@Theresa-o hold up! It appears our designer @benenright has worked on designs for this already :) I’ll share them here once I can confirm.

**Comment by Theresa-o** [comment]
@thibaudcolas 

Ok

**Comment by benenright** [comment]
Hi

Here is a simple solution to this issue. Utilising the same underline pattern we use for active tabs. I don't envisage this line would appear on hover, the hover effect would stay the same as is.


![Image](https://user-images.githubusercontent.com/4907819/229500364-6e2ec4f3-6486-41cc-a017-29f7133348f0.png)

Cheers
Ben
